OpenOversight/app/main/views.py

```diff
@@ -667,6 +667,8 @@ def label_data(department_id=None, image_id=None):
                          .filter(Face.original_image_id == form.image_id.data).first()
         if not officer_exists:
             flash('Invalid officer ID. Please select a valid OpenOversight ID!')
+        elif department and officer_exists.department_id != department_id:
+            flash(f'The officer is not in {department.name}. Are you sure that is the correct OpenOversight ID?')
```

Comment:
Flake8 fails here because python 3.5 does not support the "f-string" string formatting. I am not sure if we still use python 3.5 in production or anywhere, so I am definitely in favor of dropping 3.5 support, but until we do you might need to use `.format`

Reply:
Thank you for the pointer! I've updated that :)